week-3/Homework/mandatory/2-bush-berries.js

```diff
@@ -10,9 +10,11 @@
   Use the tests to confirm which message to return
 */
 
-function bushChecker() {
-
+function bushChecker(bushes) {
+  let bushBerryColours = bushes.every(Element => Element === "pink");
```

Comment:
Just a little thing, but remember that we always name variables with `camelCase`. So the first letter should always be lowercase, then every word after that starts with an uppercase letter.